tinybmp/Cargo.toml

```diff
@@ -18,6 +18,20 @@ exclude = [
 [badges]
 circle-ci = { repository = "jamwaffles/embedded-graphics", branch = "master" }
 
+[[test]]
+name = "embedded_graphics"
+required-features = ["postgres", "tools"]
```

Comment:
I didn't know that e-g has database support :wink: Did you maybe copy this from somewhere else?

Reply:
Oh my dog what was that about. Turns out this was also hiding the broken tiny* tests, so I've fixed those up now as well.